keda/templates/manager/minimal-rbac.yaml

```diff
@@ -34,6 +34,10 @@ rules:
   verbs:
   - create
   - update
+{{- if .Values.permissions.operator.restrict.secret }}
+  - list
+  - watch
+{{- end }}
```

Comment:
this is little bit counter intuitive, but this means that if the secret access should be restricted, it's restricted [here](https://github.com/kedacore/charts/blob/6aa60122e1dbd7d2ccf0a904eda677a01ae9fc05/keda/templates/manager/clusterrole.yaml#L33-L50), but the operator needs list & watch at least for the `.Release.Namespace` (defaults to `keda`)